### ISSUE TITLE
Пустой основной файл

### ISSUE THREAD

**Issue by DaSh-More** [blyamur/VK-Video-Download#2]
Файл vk_video_download.py содержит только три единички, и, очевидно, не работает

**Comment by eremeyko** [comment]
Не работает от слова совсем
Тестил на этих видео: https://vk.com/video-181649388_456239020,https://vk.com/video-158251423_456240525,https://vk.com/video-165454279_456239084,https://vk.com/video-158251423_456240766 (скачается 
 корректно только "ДОРАДУРА")
 (для обхода .unknown_video добавьте сразу 'outtmpl': 'downloads/%(title)s.mp4'. ВК только Mp4 качает. 
 Переименование на битость файлов не влияет!)

**Comment by eremeyko** [comment]
Это очень странно, потому что у меня пусть всё и скачано, но ни через прямой запуск, ни через venv не сохраняется целостность файлов. Буду копаться, разбирать и менять под себя тогда, спасибо

**Comment by C0la149** [comment]
@blyamur 
Привет! Использую программу эту чтобы качать видео с плейлиста. В начале все пошло отлично, с большой скоростью. Прям все выкачивал все быстро. Потом спустя некоторое время скорость прям идеально упала. Есть ли решение на это?
![изображение](https://github.com/blyamur/VK-Video-Download/assets/64633686/545c3562-ad38-4212-8700-f1678d255f8b)
Качаю видео слишком много.
Вот ссылки.
https://vk.com/video/playlist/557270913_22
https://vk.com/video/playlist/557270913_21
https://vk.com/video/playlist/557270913_23

**Comment by C0la149** [comment]
> > yt-dlp последняя версия на момент 27.05.2024
> 
> Режет вк, это наиболее вероятно, другие сервисы насколько слышал делают так же. Касательно антивируса, тоже логично, он всегда срабатывает на запакованные файлы. Для успокоения через онлайн сервисы можно файл прогнать и добавит в исключение, потом запустить.

Понял. А насчет обрезки скорости. Возможно ли обойти это? К примеру в мультепотоке качать?

**Comment by dzhizhkin** [comment]
Извиняюсь за оффтоп, а скрипт работает со Stories?

**Comment by eremeyko** [comment]
> Извиняюсь за оффтоп, а скрипт работает со Stories?

К сожалению, библиотека yt-dlp, на которой работает приложение, не поддерживает скачивание историй
![image](https://github.com/user-attachments/assets/3c4e1f1d-d90d-4959-95a1-6ccfcd61d818)

**Comment by yarodevuci** [comment]
а рутуб качает?

**Comment by yarodevuci** [comment]
> > @blyamur а рутуб качает?
> > и еще вопрос VPN нужно включать чтобы качать видео которые только для RU?
> 
> YouTube качает. Rutube надо проверить. В общем то основная задача была, качать с вк

да, просто интересно, а то народ пишет что качает не только с ВК :D

**Comment by yarodevuci** [comment]
вау рутуб тоже качает! 
автору респект вообще!!!

**Comment by yarodevuci** [comment]
@blyamur нужно добавить в ReadMe что не только ВК видео поддерживает ))))

**Comment by eremeyko** [comment]
Может вопрос закроем?)
